common/reflectivity/reflectivity.h

```diff
@@ -25,11 +26,14 @@ namespace rs2
         // Reset depth samples history (queue history will reset only if needed (if add_depth_sample was called since last reset/construction)
         void reset_history();
 
+        // Return the samples history ratio samples_count / capacity [0-1]
+        float get_samples_ratio() const;
+
     private:
         // Implement a cyclic queue for depth samples
         std::deque<float> _dist_queue; 
 
-        // Use to indicate reset() that the queue contain samples
-        std::atomic_bool _is_empty;
+        // Holds the current samples_count
+        int _samples_count;
```

Comment:
So... this is more of a "history size"? Can `get_samples_ratio` be `size() / capacity()`, then? Do you think it's clearer?

Reply:
I changed the name to _history_size,

Did you mean add 2 new functions called size() & capacity()?

I think it's clear as is.. I updated the function comment to:

// Return the samples history ratio size / capacity [0-1]